src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private final int firstLeg;
+    private final int secondLeg;
+
+    public RightTriangle(String color, int firstLeg, int secondLeg) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double getArea() {
+        return (firstLeg * secondLeg) * 0.5;
+    }
+
+    @Override
+    public void toDrawFigure() {
```

Comment:
Don't use verbs for class/interface names. The method 'toDrawFigure' should be named in a way that it represents an action performed by the object, such as 'draw'.